coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/sql/SqlSecondaryManifest.scala

```diff
@@ -396,9 +397,12 @@ class SqlSecondaryManifest(conn: Connection) extends SecondaryManifest {
       try {
         stmt.executeUpdate()
       } catch {
+        case PostgresUniqueViolation(_*) =>
+          conn.rollback(savepoint)
+          throw ResyncLaterSecondaryException("wait for another resync")
         case ex: SQLException =>
           conn.rollback(savepoint)
-          throw ex
+          throw ResyncLaterSecondaryException(ex.getMessage)
       }
```

Comment:
Are we sure we don't want to catch all exceptions and throw a `ResyncLaterSecondaryException` here?